app/contracts/work_packages/base_contract.rb

```diff
@@ -80,6 +80,10 @@ class BaseContract < ::ModelContract
     end
 
     attribute :schedule_manually
+    attribute :ignore_non_working_days,
+              writeable: ->(*) {
```

Comment:
All for it, and don't care for which one it turns out to be.

Reply:
`writable` is the right syntax to pick.

I even lost 2 hours while working on the file links feature because the wrong syntax was used at the wrong place and it took me a while to figure it out. Then I wrote a rubocop to detect wrong usages but never commited it because I could not add tests for it, and I was not sure if that would be the right thing to do...

Reply:
Writable is the predominant one.

Reply:
All right, then let's rename everything to `writable` but let's do it in a separate PR.